src/main/java/io/seqera/tower/cli/responses/HealthCheckResponse.java

```diff
@@ -0,0 +1,34 @@
+package io.seqera.tower.cli.responses;
+
+import io.seqera.tower.cli.utils.TableList;
+
+import java.io.PrintWriter;
+
+public class HealthCheckResponse extends Response{
```

Comment:
To use this command inside a bash script, it's interesting to return an `exitCode` equal to -1 when there is at least one fail (otherwise return a zero).

Currently it is not possible to set the exit code of a response, but we can add a `getExitCode` method to the abstract class `Response` that by default returns a zero and override it here.